LayoutTests/css3/calc/inner-border-radius-longer-than-width-expected.txt

```diff
@@ -0,0 +1,6 @@
+layer at (0,0) size 800x600
+  RenderView at (0,0) size 800x600
+layer at (0,0) size 800x600
+  RenderBlock {HTML} at (0,0) size 800x600
+    RenderBody {BODY} at (8,8) size 784x584
+      RenderBlock {DIV} at (0,0) size 300x243 [bgcolor=#B8860B] [border: none (100px solid #FF0000) (43px solid #000000) none]
```

Comment:
Please don't create new render tree dump based tests, they are annoying to maintain as they are very sensitive to unrelated changes.

Instead this should likely be a reftest where you add an -expected.html file that the test result gets pixel-compared against. For non-visual things a text based test (testRunner.dumpAsText()) can be used.